observatorium/overlays/moc/smaug/loki/obcs/opf-observatorium-metrics.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: objectbucket.io/v1alpha1
```

Comment:
already seems to be exisiting:
https://github.com/operate-first/apps/blob/3c05ac9fb559f83e6178d47ab16a4b47aa759769/observatorium/overlays/moc/smaug/thanos/obcs/opf-observatorium-metrics.yaml